docs/NewOBOFC

```diff
@@ -0,0 +1,19 @@
+---
+layout: doc
+id: OFOCMember
+title: Becoming a member of the OBO Operations Committee
+---
+
+# Becoming a member of the OBO Foundry Operations Committee (OFOC)
+
+Note: This is _not_ a formal process, but rather a piece of documentation to make the _informal_ process more transparent. 
```

Comment:
to make the _informal_ process ->
to make the current _informal_ process

(to make it clear that it is what we're doing now but that it will change)

Reply:
ok